File: demo-wiremock/src/test/java/io/fries/wiremock/demo/wiremock/CucumberTestRunner.java
package io.fries.wiremock.demo.wiremock;

import io.cucumber.junit.Cucumber;
import io.cucumber.junit.CucumberOptions;
import org.junit.runner.RunWith;
import org.springframework.context.annotation.ComponentScan;

@RunWith(Cucumber.class)
@CucumberOptions(
        glue = "io.fries.wiremock.demo.wiremock.cucumber",
        features = "src/test/resources/features",
        plugin = {"pretty", "html:target/cucumber/cucumber.html", "json:target/cucumber/cucumber.json"}
)
public class CucumberTestRunner {
}

File: examples/demo-cucumber/src/test/kotlin/io/fries/proxito/demo/cucumber/CucumberTestRunner.kt
package io.fries.proxito.demo.cucumber

import io.cucumber.junit.Cucumber
import io.cucumber.junit.CucumberOptions
import org.junit.runner.RunWith

@RunWith(Cucumber::class)
@CucumberOptions(
    glue = ["io.fries.proxito.demo.cucumber.glue"],
    features = ["src/test/resources/features"],
    plugin = ["pretty", "html:target/cucumber/cucumber.html", "json:target/cucumber/cucumber.json"]
)
class CucumberTestRunner